packages/editor-ui/src/mixins/expressionManager.ts

```diff
@@ -83,14 +86,18 @@ export const expressionManager = defineComponent({
 				throw new Error(`Failed to parse expression: ${this.editor.state.doc.toString()}`);
 			}
 
+			const skipSegments = ['Program', 'Script', 'Document', ...this.skipSegments];
+
 			fullTree.cursor().iterate((node) => {
-				if (node.type.name === 'Program') return;
+				const text = this.editor.state.sliceDoc(node.from, node.to);
+
+				if (text === '' || skipSegments.includes(node.type.name)) return;
```

Comment:
Do we still need to check for an empty string now that we are exposing the `Whitespace` token? Or this is here for another reason?

Reply:
Not any more. Removing this.